ports/nrf/mpconfigport.h

```diff
@@ -44,8 +44,8 @@
 #endif
 #define MICROPY_ALLOC_PATH_MAX      (512)
 #define MICROPY_PERSISTENT_CODE_LOAD (0)
-#define MICROPY_EMIT_THUMB          (0)
-#define MICROPY_EMIT_INLINE_THUMB   (0)
+#define MICROPY_EMIT_THUMB          (1)
+#define MICROPY_EMIT_INLINE_THUMB   (1)
```

Comment:
These should not be enabled for all boards by default, it's better to just define them in `mpconfigboard.h` if that board has enough space (like nrf52840).

Since the defaults for these are "disabled" the best way forward here would be to remove these 2 lines from this file, and enable the feature in `mpconfigboard.h`.

Reply:
Yeah, i really though so....
So, should i remove these from the mpconfigport and add it to all the mpconfigboard of the nrf boards for not loosing compatibility?

Reply:
Yes remove them from `mpconfigport.h` (this won't change anything because disabled is the default), and then enable (add) only to this new board that you define in this PR.